module/VuFind/src/VuFind/ILS/Logic/RenewalsHelper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * VuFind Helper - Renewals Support Methods
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
Might as well update this:

```suggestion
 * Copyright (C) Villanova University 2010-2025.
```